examples/shap/README.md

```diff
@@ -0,0 +1,9 @@
+# SHAP Examples
+
+Examples for the SHAP integration (`mlflow.shap`).
```

Comment:
```suggestion
Examples demonstrating use of the ``mlflow.shap`` APIs for model explainability
```

I was also going to say we should include a link to the ``mlflow.shap`` API docs, but since the docs aren't up yet, we should wait to do so until after the next release